.github/workflows/pull-request.yml

```diff
@@ -20,41 +20,42 @@ env:
           
 jobs:
   Linter:
-    name: Lints the code
+    name: Check for linting errors
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - name: Install Dependencies
         run: npm ci
+      - name: Run ESLint to check for linting errors
+        run: npm run lint:check
 
-      - name: Run ESLint and Prettier
-        run: npm run lint
-        
+  Type-Checker:
+    name: Checks the code for type errors
+    runs-on: ubuntu-latest
+    needs: Linter
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: npm ci
```

Comment:
1. Put the type checker in `Linter`like we did in Talawa-adm in.
1. Remove this job and references to it

Reply:
Okay!